src/middleware/persist.ts

```diff
@@ -119,6 +119,16 @@ export interface PersistOptions<S, PersistedState = S> {
    * By default, this function does a shallow merge.
    */
   merge?: (persistedState: unknown, currentState: S) => S
+
+  /**
+   * An optional boolean that will prevent the persist middleware from triggering hydration on initialization,
+   * This allows you to call `rehydrate()` at a specific point in your apps rendering life-cycle.
+   *
+   * This is useful in SSR application.
+   *
+   * @default undefined
```

Comment:
I know that _technically_ it is undefined, but I think it would be more understandable with `false`.

Please don't change it if you disagree, though.

Reply:
```suggestion
   * @default false
```